tslearn/hdftools.py

```diff
@@ -5,7 +5,7 @@
 from warnings import warn
 
 
-def save_dict(d, filename, group, raise_type_fail=True):
+def save_dict(d, filename, group, raise_type_fail=True, append=False):
```

Comment:
Will this append work? Will it always deserialize or can you only append specific things? Also, a small update in the docs could be useful here (the `FileExistsError` will only be raised if append=False and the path does already exist.

Reply:
Not sure, I implemented it while trying to add `to_hdf5` for SahepeltModel, but I finally did not succeed, so I can remove it.